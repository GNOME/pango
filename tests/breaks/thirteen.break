# some line breaks remain in words
a <span segment='word'>ab​sp​  de­fg</span> b
