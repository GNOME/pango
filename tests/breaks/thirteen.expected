Text:         ⁦a⁩  [ ]  ⁦a⁩ ⁦b⁩ [0x200b]  ⁦s⁩ ⁦p⁩ [0x200b] [ ] [ ]  ⁦d⁩ ⁦e⁩ [0xad]  ⁦f⁩ ⁦g⁩  [ ]  ⁦b⁩  [0x0a]  
Breaks:     c  c    lc c c        lc c c        c   c   lc c c      lc c c    lc c       Lc
Whitespace:    x                                x   x                    x       w       w 
Sentences:  bs                                                                   e       b 
Words:      bs be   bs                                                   be   bs be      b 
Graphemes:  b  b    b  b b        b  b b        b   b   b  b b      b  b b    b  b       b 
Hyphens:               i i           i i                   i i      i  i                   
